_posts/2017-10-13-jb-in-da-house.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title:  "J.B. Rainsberger in da house"
+date: 2017-10-13 14:03:00 +0200
+categories: learn product 
+author_name: Wolfram Kriesing
+author_url : /author/wolframkriesing
+author_avatar: wolframkriesing
+show_avatar : true
+read_time : 3
+show_related_posts: false
+square_related: recommend-wolf
+feature_image: posts/2017-10-13-jbhc17/poster.jpg
+---
+
+It started in May, that we thought about inviting [J.B. Rainsberger][jbrains] to HolidayCheck. Yeah, there was some bias from my side, since I had the pleasure to have learned from him a couple years ago. So I was all in. I just had to convince my colleagues and the company to pay for it. And it went quite smooth. HolidayCheck is quite committed to improve continuously.
+
+[jbrains]: https://twitter.com/jbrains
+
+## 7:26min
+
+If you have never heard of “7 minutes 26 seconds”? Now you did! It is a short and inspiring [lightning talk][7-26-talk] by J.B. called “7 minutes, 26 seconds, and the Fundamental Theorem of Agile Software Development”. Here at HolidayCheck people refer to it as the “Accidental Complication“ talk. I think the talk itself inspired many people here to be as excited about having J.B. around. 
```

Comment:
Talk is being mentioned probably too many times. Perhaps if you skip `talk` after `Accidental Complication` it will look better.

Reply:
done, thx